src/statistikk/Svartelling.tsx

```diff
@@ -1,38 +1,73 @@
 import React, { FunctionComponent } from 'react';
 import { Dialog, DialogSuccess, SpeechBubble } from '@navikt/ds-icons';
+import css from './Svartelling.module.css';
+import { Detail, Label } from '@navikt/ds-react';
 
 export enum SvartellingIkon {
     Delt,
     Ja,
     Nei,
-    Ubesvart,
+    SvarteIkke,
 }
 
 type Props = {
     svartellingIkon: SvartellingIkon;
-    antall: number;
-    tellingtekst: string;
+    oppsummering: string;
+    detaljer: string;
+    forklaring: string;
 };
 
-const Svartelling: FunctionComponent<Props> = ({ svartellingIkon, antall, tellingtekst }) => {
+const svgX = (
```

Comment:
Jeg ville flytta denne til en egen fil! En bonus hvis du gjør det er at du kan lagre den som en svg-fil, så importere den som en React-komponent:

```
import { ReactComponent as Kryss } from './kryss.vg';

...
<Kryss />
...
```

Docs: https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs

Reply:
fikser nå